actions/kustomize-gke-deploy/action.yaml

```diff
@@ -15,8 +15,10 @@ runs:
     - name: Deploy kustomized crds
       shell: bash
       run: |
-        kubectl kustomize ${{ inputs.kustomization-path }} | kfilt -i kind=CustomResourceDefinition | kubectl apply -f -
-        kubectl kustomize ${{ inputs.kustomization-path }} | kfilt -i kind=CustomResourceDefinition | kubectl wait --for condition=established --timeout=${{ inputs.timeout }}s -f -
+        if [ ! -z "$(kubectl kustomize ${{ inputs.kustomization-path }}  | kfilt -i kind=CustomResourceDefinition)" ]; then
```

Comment:
I dont really understand what we are checking with the `if` condition

Reply:
We are checking if the `kubectl kustomize ${{ inputs.kustomization-path }} | kfilt -i kind=CustomResourceDefinition` returns anything. Before we always applied everything returned by that command, which failed if there were no CustomResourceDefinitions to deploy